src/components/organisms/upload-modal/index.tsx

```diff
@@ -250,8 +250,11 @@ function UploadModal(props: UploadModalProps) {
             name="medusa-template-product-list.csv"
             size={624220000}
             action={
-              // TODO: download actual file on click
-              <a className="w-6 h-6 cursor-pointer" onClick={console.log}>
+              // TODO: download actual file on click - implement template creation in core
```

Comment:
todo: I think we should remove this section for now as it causes more harm than good. The template file is of high priority - will create an issue for it

Reply:
I added a static file as a temp fix until we add support for generating one server side, is that ok?

Reply:
That's awesome! 💪